apps/google-analytics-4/lambda/.env

```diff
@@ -1,6 +1,6 @@
 # Important! These values should only be used when running tests. All environment variables must be
 # specified in serverless.yml or they will not function in the real Lambda environment
-SIGNING_SECRET: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+SIGNING_SECRET: QPFQDCfipitJcFGb4f-xnP0RhZOqnStP2thYUkqg_CcWdTlFWu4KCP6pYrWPto5x
```

Comment:
was this intentional? Or does this need to revert back to a placeholder?

Reply:
it is intentional, it's a random signing secret from my own random app - no security concerns